stablehlo/reference/Element.cpp

```diff
@@ -350,6 +350,33 @@ Element min(const Element &e1, const Element &e2) {
       });
 }
 
+// These cases are implementation-defined, and the behavior of this interpreter:
+// For integers: if rhs is 0, the result is the same as `lhs`.
+//   For signed integers, if lhs is INT_MAX and rhs is -1, the result is 0.
+// For floats: use APFloat.mod(). If rhs is 0, the result is 0x7FF8000000000000
+Element rem(const Element &e1, const Element &e2) {
+  return map(
+      e1, e2,
+      [&](APInt lhs, APInt rhs) {
+        if (rhs == 0) return lhs;
+        return isSupportedSignedIntegerType(e1.getType()) ? lhs.srem(rhs)
+                                                          : lhs.urem(rhs);
+      },
+      [](bool lhs, bool rhs) -> bool {
+        llvm::report_fatal_error("reminder of two bool numbers is unsupported");
```

Comment:
Let's use the wording similar to other functions above: "rem(bool, bool) is unsupported"?